sixpack/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.2'
+__version__ = '2.1.0'
```

Comment:
Version changes should not be addressed in a PR. See the README in the main repository folder.